makefiles/tools/serial.inc.mk

```
@@ -53,7 +53,13 @@ else ifeq ($(RIOT_TERMINAL),picocom)
   TERMPROG  ?= picocom
   TERMFLAGS ?= --nolock --imap lfcrlf --baud "$(BAUD)" "$(PORT)"
 else ifeq ($(RIOT_TERMINAL),miniterm)
-  TERMPROG  ?= miniterm.py
+  # Check if miniterm.py is available in the path, if not use just miniterm
+  # since new versions will only have miniterm and not miniterm.py
+  ifeq (, $(shell command -v miniterm.py 2>/dev/null))
```

```suggestion
  ifeq (,$(shell command -v miniterm.py 2>/dev/null))
```

I think otherwise this will never trigger, as `""` is not the same as `" "` (empty string vs string with a space only in it).

---

Works both ways but I did the change.